contracts/RelayHub.sol

```diff
@@ -138,7 +134,7 @@ contract RelayHub is IRelayHub {
             initialGas >= maxPossibleGas,
             "Not enough gas left for innerRelayCall to complete");
 
-        uint256 maxPossibleCharge = calculateCharge(
+        uint maxPossibleCharge = calculateCharge(
```

Comment:
uint256 for consistency?

Reply:
@drortirosh you resolved issue but did not change code